x/evidence/types/evidence.go

```diff
@@ -87,7 +87,7 @@ func (e Equivocation) GetTotalPower() int64 { return 0 }
 
 // FromABCIEvidence converts a Tendermint concrete Evidence type to
 // SDK Evidence using Equivocation as the concrete type.
-func FromABCIEvidence(e abci.Evidence) exported.Evidence {
+func FromABCIEvidence(e abci.Misbehavior) exported.Evidence {
```

Comment:
does abci.Evidence -> abci.Misbhavior change come from dependency update for abci?

Reply:
Breaking change in Tendermint v0.37 :/